### DISCUSSION TITLE
Sample Dataset: Cape Hatteras Landsat8 Only reaches 0.778 val_mean_iou after finishing runing train_dataset

### DISCUSSION THREAD

**Discussion by bestplanetarian** [Doodleverse/segmentation_gym#146] [Q&A]
Hi Sir:

I follow the instruction and successfully install segmentation gym. Although my python version is 3.9 rather than 3.10.

I did follow the guide to create train validation dataset. Then I run the model, and change the "mixed_float16" to "float32" in "train_model.py" because of my cuda compatibility is low.

After finishing traing, I receives  0.778 val_mean_iou, which is lower than val_mean_iou: 0.9843 shown your website:
https://github.com/Doodleverse/segmentation_gym/wiki/02_Case-Study-Demo

Even though I change it back to "mixed_float16", which causes program running very slowly, this problem does not solve.

What should I do

**Comment by ddbuscombe** [comment]
Oh, interesting. I can look into that at some point and update the documents. Thanks for drawing my attention to it.

This is a toy problem and the model is not really supposed to be used in application. After all, it is trained on only a handful of images at one site. Naturally, it will be useless outside of its intended purpose. The purpose of the test dataset is to test the functionality of the program. It seems like you have successfully managed to get the model to train, so at this point, I would advise you just use the program for your own purposes on your own data.
